### ISSUE TITLE
longer display time/toggle?

### ISSUE THREAD

**Issue by alien-sunset** [Loedje/jukebox_action_bar_fix#1]
I love the vibrant colors! so nice!

is it possible to allow for a display time longer than "999"
or maybe a toggle for when the display goes away and you need it back or if you were just out of range for it to display when the music started?

**Comment by Loedje** [comment]
Thanks! You mean the action bar constantly being displayed if you are in
range of the jukebox?

**Comment by alien-sunset** [comment]
yes! exactly that! or if you need to sleep during a song, the "sleeping through the night" overrides the song title even when you are in range. so having a toggle to bring back what the music was would be cool.
